docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -35,17 +35,17 @@ The main features of the Sensor API include:
 
   **Table: Supported sensor types**
 
-  | Supported sensor types                   |                                          |                                          |
-  |------------------------------------------|------------------------------------------|------------------------------------------|
-  | [Accelerometer](#accelerometer)          | [Heart Rate Monitor Sensor](#hrm)        | [Proximity Sensor](#proximity)           |
-  | [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Humidity Sensor](#humidity)             | [Rotation Vector Sensor](#rotation)      |
-  | [Gravity Sensor](#gravity)               | [Light Sensor](#light)                   | [Significant Motion Sensor](#significant_motion) |
-  | [Gyroscope](#gyro)                       | [Linear Acceleration Sensor](#lin_accelerometer) | [Sleep Monitor](#sleep_monitor)          |
-  | [Gyroscope Rotation Vector Sensor](#gyro_rotation) | [Magnetic Sensor](#magnetic)             | [Temperature Sensor](#temperature)       |
-  | [Heart Rate Monitor LED Green Sensor](#hrm_green) | [Orientation Sensor](#orientation)       | [Ultraviolet Sensor](#ultraviolet)       |
-  | [Heart Rate Monitor LED IR Sensor](#hrm_ir) | [Pedometer](#pedometer)                  | [Uncalibrated Gyroscope](#uncal_gyro)    |
-  | [Heart Rate Monitor LED Red Sensor](#hrm_red) | [Pressure Sensor](#pressure)             | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-
+  |                                                               |                                                  |                                                  |
+  |---------------------------------------------------------------|--------------------------------------------------|--------------------------------------------------|
+  | [Accelerometer](#accelerometer)                               | [Heart rate monitor LED IR sensor](#hrm_ir)      | [Proximity sensor](#proximity)                   |
```

Comment:
@mijin85cho, this table is just a list of all sensors, but I got a bit confused when I saw them listed across 3 columns. Can this be listed in single column the list could run long, thoughts?

Reply:
@annie-samsung If we change the table to a single column list, the expected view would be like this. 
![image](https://user-images.githubusercontent.com/9958374/104142366-9e71f880-53fe-11eb-854d-49bc2ac4630d.png)

Indeed the table was to support the fast access to each supported sensor type description. 
However, as now docs.tizen.org provides the ToC on the right, 
if we consider changing the table to a single column list, I not sure with the efficiency as because of the length of the list and existence of the similar thing (ToC) it can be a burden to the layout.

Reply:
@mijin85cho, I agree with your views.
@bbrto21, there is blank row at the top (header row may be), kindly remove it.

Reply:
@annie-samsung if I remove header row, the markdown format will be broken

Reply:
@bbrto21 We do agree that GitHub flavoured Markdown doesn't support MD tables without headers. 
We have two choices here: 
1.As @annie-samsung suggested, to remove the header row, probably we can use HTML to control table attributes. 
2. To keep the table as is. 

@mijin85cho & @bbrto21 what is your viewpoint on this?

Reply:
@nawab-samsung
I think that we can remove ``` **Table: Supported sensor types**``` simply as another alternative. that is unnecessary 
So I suggest below,  I think it looks good to me.
 * Remove  ``` **Table: Supported sensor types**```
 * Remove all newlines between the note and the table
 * Leave the table as it is.

Reply:
@bbrto21 sure, let you push the changes as per your opinion.

Reply:
@nawab-samsung I updated this PR

Reply:
@bbrto21 Thanks